src/VisualStudio/Core/Def/ProjectSystem/FileChangeWatcher.cs

```diff
@@ -252,21 +252,15 @@ public static WatcherOperation CombineRange(ImmutableSegmentedList<WatcherOperat
                 {
                     case Kind.WatchFiles:
                         for (var i = 0; i < op._paths.Count; i++)
-                        {
                             fileNamesBuilder.Add(op._paths[i]);
```

Comment:
is this not just .AddRange (and the same for the below lines)?

Reply:
op._paths is a OneOrMany, I don't think there is an override for ArrayBuilder.AddRange that takes that in.